php-frontend/src/main/java/org/sonar/plugins/php/api/tree/expression/ListExpressionTree.java

```diff
@@ -36,7 +37,9 @@
 
   SyntaxToken openParenthesisToken();
 
-  SeparatedList<ExpressionTree> elements();
+    List<Optional<ArrayAssignmentPatternElementTree>> elements();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)